packages/sdk/chainDeploy/mainnets/fantom.ts

```diff
@@ -99,7 +182,27 @@ export const deploy = async ({ run, ethers, getNamedAccounts, deployments }: Cha
 
   // Liquidators
 
-  ////
+  //// CurveLPLiquidator
+  const curveLpTokenLiquidatorNoRegistry = await deployments.deploy("CurveLpTokenLiquidatorNoRegistry", {
```

Comment:
should we also deploy and configure the two curve LP token oracles that the bot would expect to have been deployed?

Reply:
Which two curve LP oracles? Curve V1 and curve V2?